### DISCUSSION TITLE
[Showcase] Flying bees

### DISCUSSION THREAD

**Discussion by pgilfernandez** [friction2d/friction#307] [Show and tell]
Just another little project to get used to Friction in a real project. This time it is not as "cookie" and polish as the Rolly one but helped me to get a little bit deeper into the software. BTW, I'm still in love with it 😄

https://github.com/user-attachments/assets/9ed42a41-fc90-479b-a60d-d22beb361b24

**Comment by NusryNizam** [comment]
hahaha, this looks nice!
